bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+
+The BIPs repository is focused on information and technology that supports the bitcoin currency.[^astroturfing]
+
+## BIP format and structure
+
+### Specification
+
+BIPs should be written in mediawiki or markdown format.
+
+Each BIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short (~200 word) description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe alternate designs that were considered and related work. The rationale
+  should address important objections or concerns raised during discussion.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any Bitcoin project to create an interoperable implementation.
+* Backwards compatibility -- BIPs that introduce backwards incompatibilities must include a section describing these
+  incompatibilities and their severity. The BIP must explain how the champion proposes to deal with these
+  incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors and documentation must be
+  completed before the BIP can be given status "Nominated". It is recommended to focus on specification and rationale
+  before writing code.
+* Footnotes -- A collection of footnotes cited in the BIP, and a place to list non-inline hyperlink targets
+* Copyright -- The BIP must be placed under an acceptable license ([see BIP Licensing below](#bip-licensing)).
+
+Some sections may not be necessary for all BIPs. When in doubt, a section should be included and briefly state how its
+topic has been sufficiently addressed.
+
+#### BIP header preamble
+
+Each BIP must begin with an RFC 822 style header preamble. The headers must appear in the following order. Headers
+marked with "\*" are optional. All other headers are required.
+
+```
+  BIP: <BIP number, or "?" before being assigned>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title; maximum 44 characters>
+  Champions: <list of champions’ names and email addresses>
+  Status: <Draft | Nominated | Active | Abandoned>
+  Type: <Standards Track | Informational | Process>
+  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+  License: <abbreviation for approved license(s)>
+* License-Code: <abbreviation for code under different approved license(s)>
+* Post-History: <list of mailing list threads, or other noteworthy discussions in "date: link" format>
```

Comment:
"Discussion" rather than Post-History?

Reply:
I was hoping to not change the Preamble too much, but that would be a better term. Will revisit next week.

Reply:
Sure, I replaced the Post-History header with Discussion